### DISCUSSION TITLE
How to trace for gun fire to center of the screen?

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Sixze/ALS-Refactored#339] [Q&A]
I made a Replicated variable for the camera component and assigned it. But on the traces it show the trace starting between the player feet. This is where it is in the blueprint. I need to track where the camera actually is and it's Forward Vector.

**Comment by DimidStudio** [comment]
You can try GetController->GetConrolRotation(). Then you can find forward vector from ratation
